include/cadmium/engine/pdevs_dynamic_simulator.hpp

```diff
@@ -66,12 +65,12 @@ namespace cadmium {
                  * @param initial_time is the start time
                  */
                 void init(TIME initial_time) override {
-                    LOGGER::template log<cadmium::logger::logger_info>(formatter::log_info_init, initial_time, _model->get_id());
+                    LOGGER::template log<cadmium::logger::logger_info, cadmium::logger::sim_info_init>(initial_time, _model->get_id());
```

Comment:
sim-> simulator :)

Reply:
Ok, added to my todo list to be made in the next iteration